src/index.html

```diff
@@ -5,10 +5,35 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="logo">
+        <img src="/images/logo.png"
+        alt="Logo">
+      </a>
+      <nav class="nav">
+        <ul class="list">
+          <li class="li_item"><a href="#apple" class="is-active">Apple</a></li>
```

Comment:
I would probably add another class just for styling `<a />` tag, instead of targetting whem via child of `<li />` elements,
I think it's more readable, don't you think?